spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -52,6 +52,7 @@
 "Apache version 2.0": Apache-2.0
 "Apache, Version 2.0": Apache-2.0
 "Apache-2.0 */ &#39; &quot; &#x3D;end --\n": Apache-2.0
+"Apache2.0": Apache-2.0
```

Comment:
I believe the mapping should go to `simple-license-mapping.yml` (as there is no space in the string to map), or?

Reply:
Indeed `SpdxDeclaredLicenseMappingTest` fails with this as it is.

Reply:
Thanks! Wasn't aware of that.